doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
+
+<a name="3.0"></a>
+## 3. Principles
+
+Any specification work created within CNTT **must** obey to set of principles specified by CNTT:
+
+* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+* [Reference Architecture Principles](../ref_arch#principles)
+
+<a name="4.0"></a>
+## 4. Technology Direction
+
+<a name="4.1"></a>
+### 4.1 IO Virtualization
+
+There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualisatin:
+
+- Para-Virtualization method (software only).
+- Direct assignment with VT-d.
+- Device Sharing with SR-IOV & VT-d.
+- Para-Virtualisation method (Hardware support).
+
+**Figure 3** below shows some of the relevant IO Virtualization techniques.
+
+<p align="center"><img src="./figures/tech_iov.png" alt="scope" title="Document Types" width="70%"/></p>
+<p align="center"><b>Figure 3:</b> Relevant IO Virtualisation Techniques</p>
+
+<a name="4.1.1"></a>
+#### 4.1.1 Para-virtualization method (software only)
+
+This is the preferred method of IO virtualisation as it provides flexibility and full abstraction of workloads from the underlying infrastructure. It usually relies on standard IO interfaces that are implemented in software.
+For Networking, there are two common networking interfaces used: virtio-net for KVM/QEMU and VMXNET for VMware.
+
+Using a standard interface for IO means that workload doesn't need to run any proprietary software drivers for specific hardware vendors and the implementation of that workload is completely agnostics of the hardware used.
+
+**Figure 4** below shows the typical components of a para-virtualised interface: 
+- frontEnd driver: The frontEnd driver is an off-the-shelf driver that runs on the workload.
+- backEnd driver: runs on the Hypervisor and is responsible of bridging standard communications coming from applications to a hardware specific ones. 
+
+This nature of this disaggregation is what gives the para-virtualised interfaces the flexibility that makes them favourable in a virtualised environment.
+
+The downside of para-virtualisation interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+
+<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 4:</b> Para-Virtualised interface components (software only).</p>
+
+<a name="4.1.2"></a>
+#### 4.1.2 Direct assignment with VT-d
+
+Direct assignment (or pass-through) is when an IO device is directly assigned to a workload by-passing the hypervisor. Direct assignment is supported in intel platforms (using VT-d technology) and by AMD platforms (using AMD-V technology) as shown in **Figure 5**.
+
```

Comment:
In Figure 3, the pass-through is shown as a bypass -- need Figures 3, 5 and 6 to be ocnsistent

Reply:
I will dig in the originals figure to fix

Reply:
Consider: "Direct Assignment is supported in x86 architectures through an IOMMU (Input/Ouput Memory Management Unit), which provides the ability for a PCIe device to autonomously (i.e. without hypervisor intervention) perform DMA transfers directly into guest memory.